src/cargo_ops/elaborate_workspace.rs

```diff
@@ -77,7 +77,7 @@ impl<'ela> ElaborateWorkspace<'ela> {
             .iter()
             .map(|feature| FeatureValue::new(InternedString::from(feature)))
             .collect();
-        let specs = Packages::All.to_package_id_specs(workspace)?;
+        let specs = Packages::All(Vec::new()).to_package_id_specs(workspace)?;
```

Comment:
Do you know what the empty vec means here?

Reply:
this is cargo refactor https://github.com/rust-lang/cargo/commit/23ab2afc25536ec49926ba9264833d78b4ad3776